addon-test-support/@ember/test-helpers/index.ts

```diff
@@ -25,6 +25,7 @@ export { default as setupOnerror, resetOnerror } from './setup-onerror';
 export { getDebugInfo } from './-internal/debug-info';
 export { default as registerDebugInfoHelper } from './-internal/debug-info-helpers';
 export { default as getTestMetadata } from './test-metadata';
+export { registerHook } from './-internal/helper-hooks';
```

Comment:
This export should probably be underscored while we figure things out, also I think we need to expose `runHooks` as well (so authors of custom helpers can participate in the same system).

```suggestion
export { registerHook as _registerHook, runHooks as _runHooks } from './-internal/helper-hooks';
```